### PR TITLE
Edit seller info form

### PR DESCRIPTION
**Pull request by Dustnclay**
# Description
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
- [ ] New feature (non-breaking change which adds functionality)
## Change Status
- [ ] Complete, but not tested (may need new tests)
# Checklist
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/sellerPages/profile/dashboard/edit/EditInfo.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  }
+
+  return (
+    <>
+      <NavBar />
+      <br />
+      <br />
+      <label>
+        Name
+        <input
```
Comment (edelveiss) [inline src/components/sellerPages/profile/dashboard/edit/EditInfo.js:48]: You can use antd library from Ant Design for styling input form. In this case we will have consistence in styling. For example:
`<Form.Item
          name="seller_name"
          rules={[
            {
              required: true,
              message: 'Please input an edited seller name',
            },
          ]}
        >
          <Input placeholder="Seller Name" />
        </Form.Item>`

**Review by edelveiss — APPROVED** [review]
Looks good!

File: src/components/sellerPages/profile/dashboard/edit/EditInfo.js (hunk 1 above)
Comment (aaronmatson1) [reply src/components/sellerPages/profile/dashboard/edit/EditInfo.js:48]: Excellent catch

**Review by aaronmatson1 — APPROVED** [review]
Excellent work. I agree with Tatiana about using and for styling on the form

**Pull request merged**